src/Kovan-DssSpell.sol

```diff
@@ -15,62 +15,74 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 pragma solidity 0.6.12;
+pragma experimental ABIEncoderV2;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
-import "lib/dss-interfaces/src/dss/GemJoinAbstract.sol";
-import "lib/dss-interfaces/src/dss/IlkRegistryAbstract.sol";
-import "lib/dss-interfaces/src/dapp/DSTokenAbstract.sol";
-
-interface L1GovernanceRelayLike {
-    function relay(address, bytes calldata, uint32) external;
-}
-
 contract DssSpellAction is DssAction {
 
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
     // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/TODO -q -O - 2>/dev/null)"
-    string public constant in_memory_of = "Jeffrey Blechschmidt";
-    string public constant override description = "Optimism test spell casting + housekeeping";
-
-    // Vote delegate proxy factory
-    address constant VOTE_DELEGATE_PROXY_FACTORY = 0x1740F3bD55b1900C816A0071F8972C201566e3a3;
-    address constant L1_GOVERNANCE_RELAY         = 0xAeFc25750d8C2bd331293076E2DC5d5ad414b4a2;
-    address constant L2_SPELL                    = 0xC88e0cDAA48FA8cA12212b157fdee617be4cBD70;
+    string public constant override description = "Add MATIC";
 
+    // TODO - enable officeHours (if needed) on mainnet
     // Turn off office hours
     function officeHours() public override returns (bool) {
         return false;
     }
 
-    function actions() public override {
+    address constant MATIC                 = 0x688E1A8830Ea8dd8fe389FA2228997C663b3807A;
+    address constant MCD_JOIN_MATIC_A      = 0x4Af8801fbDD5ae4FDe2cbC9F844b09c6777525CE;
```

Comment:
Hmm I think we should deploy the `GemJoin` with solc 0.5.12. That was we've been using lately for that contract.

Reply:
If it is much burden to change it now let's leave it as it is as Kovan will be deprecated anyway.
I'm going to deploy the `GemJoin` with solc 0.5.12 for Goerli as the rest of collaterals.